openmc/filter.py

```diff
@@ -425,7 +425,7 @@ def get_pandas_dataframe(self, data_size, **kwargs):
         df = pd.DataFrame()
 
         filter_bins = np.repeat(self.bins, self.stride)
-        tile_factor = data_size / len(filter_bins)
+        tile_factor = int(data_size / len(filter_bins))
```

Comment:
I would add `from __future__ import division` at the top and then change this to:
```Python
tile_factor = data_size // len(filter_bins)
```